### ISSUE TITLE
[mat-select][a11y] focus state is not clear on list box elements while navigating in high contrast mode

### ISSUE THREAD

**Issue by glushkova91** [angular/components#17375]
#### Reproduction

Use StackBlitz to reproduce your issue: https://stackblitz.com/fork/components-issue

Steps to reproduce:
1. Turn on High Contrast mode
2. Navigate https://material.angular.io/components/select/overview
3. Open Basic Mat-select dropdown
4. Hover on non-selected element which provide hover/focus state and observe the result
 
![image](https://user-images.githubusercontent.com/13402897/66674830-c92c3a80-ec18-11e9-8018-c5e31401c53a.png)

 
#### Expected Behavior

What behavior were you expecting to see?
It should be possible to clearly see the focus indicator on mat-select drop down when High Contrast mode is enabled.

#### Actual Behavior

What behavior did you actually see?

Focus state is almost invisible

#### Environment

- Angular:
- CDK/Material:
- Browser(s): Chrome
- Operating System :  Chrome OS Version 77.0.3865.105 (Official Build) (64-bit)

**Comment by crisbeto** [comment]
Seems like this has been fixed. Here's what the focus indication currently looks like:
![Angular_Material_‎-_Microsoft_Edge_2020-05-26_18-09-42](https://user-images.githubusercontent.com/4450522/82924208-4de78100-9f7c-11ea-89c4-b0a59b098017.png)
